### PR TITLE
modified create-channel

### PR DESCRIPTION
**Pull request by ayrotideysarkar**

### CODE REVIEW

**Comment by ayrotideysarkar** [comment]
hey @cwarnermm and @justinegeffen can I see a preview?

**Comment by ayrotideysarkar** [comment]
> @ayrotideysarkar - Previews aren't generating because of syntax issues present in the code for the tabbed-content. Spacing is important in RST source files, and for tabbed content. You've introduced the tabbed content correctly with `.. tabs::`; however, all subsequent lines aren't yet indented. Once you indent the tab labels and double-indent as required for RST files, your previews should start generating.
> 
> Please use the [source code](https://github.com/mattermost/docs/blob/master/source/channels/favorite-channels.rst) for the [Mark channels as favorites](https://docs.mattermost.com/channels/favorite-channels.html) documentation page as a starting point and guide as you revisit your PR: image

Yup... I have just removed the tabs just want to see the normal page... I will add the tabs then

**Comment by ayrotideysarkar** [comment]
Hey @cwarnermm can you re run the preview?

File: source/channels/create-channels.rst [hunk 1]
```diff
@@ -17,27 +17,33 @@ Anyone can create public channels, private channels, direct messages, and group
...
-    Alternatively, you can select the **+** symbol at the top of the channel sidebar, then select **Open a Direct Message**. 
-
-In the **Direct Messages** popup, identify your most recent conversations by relative timestamps.
-
-.. image:: ../images/recent-direct-group-messages.png
-    :alt: Write a direct message or group message.
+  
+.. tabs::
+
+  .. tab:: Web browser/Desktop app
```
Comment (justinegeffen) [inline source/channels/create-channels.rst:23]:
```suggestion
  .. tab:: Web browser/desktop app
```

File: source/channels/create-channels.rst [hunk 2]
```diff
@@ -17,27 +17,33 @@ Anyone can create public channels, private channels, direct messages, and group
...
+      
+       .. image:: ../images/create-new-channel.png
+            :alt: Create a channel
+      
+     **To create a public or private channel in Web App**, select the **+** symbol at the top of the channel sidebar, then select **Create New Channel**.
```
Comment (justinegeffen) [inline source/channels/create-channels.rst:28]:
```suggestion
     **To create a public or private channel in the web app**, select the **+** symbol at the top of the channel sidebar, then select **Create New Channel**.
```

File: source/channels/create-channels.rst [hunk 3]
```diff
@@ -17,27 +17,33 @@ Anyone can create public channels, private channels, direct messages, and group
...
+
+       .. image:: ../images/write-dm.png
+            :alt: Access recent direct messages and group messages.
+   
+     **To start a direct or group message in Web App**, select the **+** symbol next to the **Direct Messages** category in the channel sidebar.
```
Comment (justinegeffen) [inline source/channels/create-channels.rst:33]:
```suggestion
     **To start a direct or group message in the web app**, select the **+** symbol next to the **Direct Messages** category in the channel sidebar.
```

File: source/channels/create-channels.rst [hunk 4]
```diff
@@ -17,27 +17,33 @@ Anyone can create public channels, private channels, direct messages, and group
...
+             Alternatively, you can select the **+** symbol at the top of the channel sidebar, then select **Open a Direct Message**.
+                 
+              In the **Direct Messages** popup, identify your most recent conversations by relative timestamps.
+                          
+                  .. image:: ../images/recent-direct-group-messages.png
+                      :alt: Write a direct message or group message.
+      
+  .. tab:: Mobile app
+   
+     **To create a public or private channel in mobile app V2**, select **+** symbol at the top right corner, then one menu will be appeared at the bottom of the screen, then select **Create New Channel**. If you want to make the channel private then enable the **Make Private** option. Otherwise it will be automatically Public Channel
```
Comment (justinegeffen) [inline source/channels/create-channels.rst:46]:
```suggestion
     **To create a public or private channel in the mobile app V2**, select the **+** symbol at the top right corner, then select **Create New Channel**. Channels are created as public by default. If you want to make the channel private select the **Make Private** option.
```

File: source/channels/create-channels.rst [hunk 5]
```diff
@@ -17,27 +17,33 @@ Anyone can create public channels, private channels, direct messages, and group
...
+                          
+                  .. image:: ../images/recent-direct-group-messages.png
+                      :alt: Write a direct message or group message.
+      
+  .. tab:: Mobile app
+   
+     **To create a public or private channel in mobile app V2**, select **+** symbol at the top right corner, then one menu will be appeared at the bottom of the screen, then select **Create New Channel**. If you want to make the channel private then enable the **Make Private** option. Otherwise it will be automatically Public Channel 
+
+    
+     **To start a direct or group message in Mobile App V2**, select  the **+** sign again, then select Open **a Direct Message** then select the person whom you want        to have a chat or you can select multiple person if you want to create a **Group**, then select **Start** . your group will be created .
```
Comment (justinegeffen) [inline source/channels/create-channels.rst:49]:
```suggestion
     **To start a direct or group message in the mobile app V2**, select  the **+** sign again, then select **Open a Direct Message**. You can select one person for a direct message or multiple people for a group message. Select **Start** to start the conversation.
```

**Review by justinegeffen — COMMENTED** [review]
Thank you! Great work. I've provided copy feedback inline. :)

**Review by justinegeffen — APPROVED** [review]

**Comment by justinegeffen** [comment]
Thanks for this contribution, @ayrotideysarkar! It's good to go. :)

**Pull request merged**